src/components/application_manager/include/application_manager/smart_object_keys.h

```diff
@@ -220,6 +220,7 @@ extern const char* is_cloud_application;
 extern const char* cloud_connection_status;
 extern const char* endpoint;
 extern const char* display_capabilities;
+extern const char* system_display_capabilities;
```

Comment:
This key isn't used anywhere

Reply:
Removed in the cc75b0ce794415e5c81fe4f1df6d780d6b0cd63e